### PR TITLE
Awsdz 46 dodanie strony z lista zwierzat

### PR DESCRIPTION
**Pull request by Zimniok**

### CODE REVIEW

File: pwa/src/components/AnimalCard.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { HtmlHTMLAttributes, HTMLProps } from "react";
+import Animal from "../models/animal.dto";
+
+function AnimalCard({ id, name, intakeDate }: Animal): JSX.Element {
+    return (
+        <a href="#!">
```
Comment (zlockibartek) [inline pwa/src/components/AnimalCard.tsx:6]: ??

File: pwa/src/components/AnimalCard.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            <div className="rounded-lg shadow-lg bg-white max-w-sm">
+                <img
+                    src="https://dummyimage.com/300x300/fff/aaa"
```
Comment (zlockibartek) [inline pwa/src/components/AnimalCard.tsx:9]: change src

File: pwa/src/models/animal.dto.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export default interface Animal {
+    id: number;
+    species: string;
+    breed: string;
+    name: string;
+    birthDate: Date;
+    intakeDate: Date;
+    description: string;
+}
```
Comment (zlockibartek) [inline pwa/src/models/animal.dto.ts:9]: EOL

**Review by zlockibartek — CHANGES_REQUESTED** [review]

File: pwa/src/components/AnimalCard.tsx (hunk 1 above)
Comment (Zimniok) [reply pwa/src/components/AnimalCard.tsx:6]: placeholder for now

File: pwa/src/components/AnimalCard.tsx (hunk 2 above)
Comment (Zimniok) [reply pwa/src/components/AnimalCard.tsx:9]: placeholder for now

**Pull request merged**